gradle-module-plugin/src/functionalTest/kotlin/io/ia/sdk/gradle/modl/BaseTest.kt

```diff
@@ -18,24 +11,29 @@ import java.nio.file.Files
 import java.nio.file.Path
 
 data class SigningResources(
-    val keystore: Path,
-    val certFile: Path,
+    val keystore: Path?,
+    val certFile: Path?,
```

Comment:
Laying the groundwork for fetching the cert file straight from the HSM. Not for all use cases, but some.